docs/_documentations/importing-existing-projects.md

```diff
@@ -186,13 +186,13 @@ Codewind works with Swift projects that use the Kitura web framework.
 Requirements:
 
 - A `Dockerfile-tools` file is generated to build the project. Ensure the project can be built by using a `release` build configuration.
-For example, you should be able to build the project by using the command
+For example, you should be able to build the project by using the command:
```

Comment:
"Should" indicates uncertainty. Unless we know this command is unstable (in which case we'd want to open an issue), avoid "should." Instead go for something like, "For example, you can build the project by using the following command:"